src/app/routes/storage.js

```diff
@@ -157,7 +157,7 @@ module.exports = (Router, Service, App) => {
       Logger.error(
         `Invalid metadata trying to create a file for user ${behalfUser.email}: ${JSON.stringify(file, null, 2)}`
       );
-      throw Error('Invalid metadata for new file');
+      res.status(500).json({error: 'Invalid metadata for new file'});
```

Comment:
As the data provided is invalid, is a [Bad Request](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400)